Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/ObjCStubs.kt

```diff
@@ -690,3 +695,8 @@ private fun genProtocolGetter(
 
     return functionName
 }
+
+private fun ObjCClassOrProtocol.isProtocolClass(): Boolean = when (this) {
+    is ObjCClass -> (name == "Protocol" || binaryName == "Protocol")
+    is ObjCProtocol -> false
```

Comment:
Interestingly, so what does it actually tests for?

Reply:
That this is `Protocol` class (not the same as protocol).
`@protocol(Foo)` has `Protocol*` type.